### PR TITLE
Use dist version of pdfjs

### PR DESCRIPTION
**Pull request by apalaniuk**
We can't avoid this when we move to Polymer 3, so may as well do this pre-conversion.

### CODE REVIEW

File: d2l-pdf-viewer.html [hunk 1]
```diff
@@ -491,6 +491,12 @@
 					type: Number,
 					value: 5
 				},
+				pdfJsWorkerSrc: {
+					type: String,
+					value: function() {
+						return this.resolveUrl('../pdfjs-dist/build/pdf.worker.min.js');
```
[the remark this answers is not in the document]
Comment (apalaniuk) [reply d2l-pdf-viewer.html]: Sure, `importPath` just doesn't seem reliable in hybrid elements under Polymer 1.x

File: d2l-pdf-viewer.html [hunk 2]
```diff
@@ -546,7 +549,8 @@
 				this.scopeSubtree(this.$.viewerContainer, true);
 
 				pdfjsLib.GlobalWorkerOptions.workerSrc =
-					'https://cdnjs.cloudflare.com/ajax/libs/pdf.js/2.0.943/pdf.worker.min.js';
+					this.pdfJsWorkerSrc ||
+					this.importPath + '../pdfjs-dist/build/pdf.worker.min.js';
```
Comment (jstefaniuk-d2l) [inline d2l-pdf-viewer.html]: will the import path always end in a `/`? Or does there need to be some logic for handling that case?

File: d2l-pdf-viewer.html [hunk 3]
```diff
@@ -546,7 +549,8 @@
 				this.scopeSubtree(this.$.viewerContainer, true);
 
 				pdfjsLib.GlobalWorkerOptions.workerSrc =
-					'https://cdnjs.cloudflare.com/ajax/libs/pdf.js/2.0.943/pdf.worker.min.js';
+					this.pdfJsWorkerSrc ||
```
Comment (jstefaniuk-d2l) [inline d2l-pdf-viewer.html]: So what's the benefit of passing this in over the fallback?

Comment (apalaniuk) [reply d2l-pdf-viewer.html]: Not a benefit, but a requirement, depending on how the client app pulls this element in.

**Pull request merged**